app/src/main/java/in/ac/bits_pilani/goa/ard/utils/AHC.java

```diff
@@ -58,4 +65,20 @@
      * background image for navigation drawer.
      */
     public static final String FDR_NAV_DRAWER_IMAGE_LIST = "navDrawerImages";
+
+    /**
+     * Method to get bitmap from vector drawable.
+     * @param context context.
+     * @param drawableId R.drawable.blah.
+     * @return bitmap.
+     */
+    public static Bitmap getBitmapFromVectorDrawable(final Context context, final int drawableId) {
```

Comment:
The same effect can be achieved using BitmapFactory methods. No need to add extra methods
`BitmapFactory.decodeResources(hostActivity.getResources, R.drawable..myicon);` returns a bitmap from a drawable